administrator/components/com_languages/access.xml

```diff
@@ -8,4 +8,12 @@
 		<action name="core.edit" title="JACTION_EDIT" description="JACTION_EDIT_COMPONENT_DESC" />
 		<action name="core.edit.state" title="JACTION_EDITSTATE" description="JACTION_EDITSTATE_COMPONENT_DESC" />
 	</section>
+	<section name="language">
+		<action name="core.permission" title="JACTION_PERMISSION" description="JACTION_PERMISSION_LANGUAGE_DESC" />
+		<action name="core.manage" title="JACTION_MANAGE" description="JACTION_MANAGE_COMPONENT_DESC" />
```

Comment:
I think the language itself doesn't need the `core.manage` acl.
I also would move the `core.permission` to be the last one.